generators/client/prompts.js

```diff
@@ -40,6 +40,10 @@ function askForClient(meta) {
         {
             value: 'angularX',
             name: 'Angular 4'
+        },
```

Comment:
since OAuth probably doesn't work with React its better to have this only in the below optional block

Reply:
OK, I can move, and then move it back once I have OAuth support working. Does JWT and Session work or just JWT?

Reply:
for now just JWT but dont worry i'lltake  care of this